lib/identity_cors.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class IdentityCors
+  FEDERALIST_REGEX = %r{^https://federalist-[0-9a-f-]+\.pages\.cloud\.gov$}
```

Comment:
Could we use `\A` and `\z` here instead of `^` and `$`? It may not matter for a URL but just to be extra safe.

Reply:
good call, will update